### ISSUE TITLE
Problem when load html footer from view in laravel 5

### ISSUE THREAD

**Issue by rafwell** [fireguard/report#7]
Hi! When load html footer from view like that
`$footer = view()->make('admin.caixas.relatorios.footer')->render();`

and pass this to Report class, i have this error: 
`The process "/mnt/rafael/fenix/projetos/fazdeconta/vendor/bin/phantomjs --debug=false --ignore-ssl-errors=true --load-images=true --ssl-protocol=any /tmp/report-script-gB4rZn /tmp/report-f9c640636fdd1a7abc8a64510c8a741517a1c554.html /tmp/report-f9c640636fdd1a7abc8a64510c8a741517a1c554.pdf" exceeded the timeout of 10 seconds.`
![image](https://cloud.githubusercontent.com/assets/2893078/21455322/82ee6bde-c906-11e6-84f6-e5512ef2e95e.png)

But, if i load the same html directly in a php var, like that, work fine:

```php
$footer = 
            '<style>  
                div{        
                    font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;    
                    font-size: 9px;
                    text-align: right;      
                    border-top: 1px #eeeeee solid; 
                    padding: 2px;
                }   
            </style>
            <div>
                Página <span>@{{ numPage }} de @{{ totalPages }}</span>
            </div>'; 
```

Thanks for this package. Him is very good.

**Comment by meneguite** [comment]
Hi @rafwell.. His view contains the same content to put in variable footer? Does not it extend some other layout that might be causing this problem? Try to give a "dd($footer);" after rendering the html and make sure it has the same content.

**Comment by rafwell** [comment]
Yes, the same content. I've copied from view. This bug is curious...

**Comment by meneguite** [comment]
You can send me the file "/tmp/report-f9c640636fdd1a7abc8a64510c8a741517a1c554.html" ?

**Comment by rafwell** [comment]
[report-f9c640636fdd1a7abc8a64510c8a741517a1c554.html.zip](https://github.com/fireguard/report/files/671119/report-f9c640636fdd1a7abc8a64510c8a741517a1c554.html.zip)

**Comment by rafwell** [comment]
The footer is not concatenated on html

**Comment by meneguite** [comment]
Hi @rafwell.. I've detected that this problem occurs because while rendering with the blade it removes @{{ turns on {{ what breaks what was expected inside the report. I'm working on a way around this problem.

**Comment by meneguite** [comment]
@rafwell this issue was resolved with the release of version 0.1.8. Please test if your problem has been resolved completely. I'm waiting for your feedback.

**Comment by rafwell** [comment]
Solved in version 0.1.8. Thanx!
